src/components/ButtonWalletPredicate/ButtonWalletPredicate.tsx

```diff
@@ -0,0 +1,44 @@
+import type { ButtonProps } from '@chakra-ui/react'
+import { Button } from '@chakra-ui/react'
+import type { InterpolationOptions } from 'node-polyglot'
+import { useCallback } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { WalletActions } from 'context/WalletProvider/actions'
+import { useWallet } from 'hooks/useWallet/useWallet'
+
+type ButtonWalletPredicateProps = {
+  isValidWallet: boolean
+  invalidWalletTranslation?: string | [string, InterpolationOptions]
+  onWalletNotConnectedClick?: () => void
```

Comment:
```suggestion
  onConnectWalletClick?: () => void
```

Reply:
q: do we need this currently or can we remove this prop? All consumers should effectively just `handleConnect()` (minus the `<AddLiquidityInput />` instance, which does more but probably shouldn't, see comment there

Reply:
I don't know why but I was feeling like it was useful, maybe the sickness killed me last week at that time

Reply:
Fixed in [1b98335](https://github.com/shapeshift/web/pull/7270/commits/1b98335766e29210cd5969426bc7506fc19efea8)